x-pack/plugin/core/src/main/resources/logs-default-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
+      },
+      "dynamic_templates": [
+        {
+          "strings_as_keyword": {
+            "mapping": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "match_mapping_type": "string"
+          }
+        }
+      ],
+      "date_detection": false,
```

Comment:
Should we have this as `true`? Could this cause problems if a date is a false positive?

Reply:
I suspect that Lee's motivation is that a value that looks like a date could make dynamic mappings create a date field, which would then fail indexing any document that has a non-date value for this field.

Reply:
should we disable `numeric_detection` too for the same reason?

Reply:
Can this be enabled / disabled per object or only globally? In general +1 on having it in to indicate if it is true or false. I'm leaning to false. @simitt might have an opinion here.